### ISSUE TITLE
Move some Digitization code from FATRAS to Core?

### ISSUE THREAD

**Issue by benjaminhuth** [acts-project/acts#2230]
During discussion with @pbutti we thought if some of the digitization code from the `FATRAS` package might better belong to `Core`. More precisely, from the Classes and files in `Fatras/Digitization` only the `UncorrelatedHitSmearer.hpp` depends on `ActsFatras::Hit`

All other files in that directory only depend on each other or on Core functionality, these are:
```
Channelizer.hpp  ChannelMerger.hpp  DigitizationData.hpp  DigitizationError.hpp  PlanarSurfaceDrift.hpp  PlanarSurfaceMask.hpp
```
as well as
```
Channelizer.cpp  DigitizationError.cpp  PlanarSurfaceDrift.cpp  PlanarSurfaceMask.cpp
```

So these don't depend on any of the FATRAS infrastructure.

Are there conceptual or design considerations for keeping this in FATRAS?

**Comment by benjaminhuth** [comment]
@paulgessinger @asalzburger @andiwand

**Comment by gagnonlg** [comment]
From what I understand (e.g. from discussions with @asalzburger), the digitization model is not very realistic, but putting it in Core would basically advertise it as "ready for production". Has the model ever been checked with actual data from a real-world detector?

**Comment by paulgessinger** [comment]
To be honest I always liked this separation of code specific to simulation from the core package.

**Comment by benjaminhuth** [comment]
Yeah I see the point. The specific reason was, that @pbutti wants to use this functionality in LDMX and it would be more convenient in Core... But it is also possible to install FATRAS, right?

We also thought about this, because there is already a Digitization package in Core, but it seems to be largely unused (most of the code seems to be last updated with the initial clang-format 6 years ago)...
Do you know if this code is in use?

**Comment by paulgessinger** [comment]
Yes. This was the idea, FATRAS is sort of a sibling to Core.
